src/pages/studyView/table/FixedHeaderTable.tsx

```diff
@@ -0,0 +1,161 @@
+    initDataStore() {
+        this._store = new LazyMobXTableStore<T>({
+            columns: this.props.columns,
+            data: this.props.data,
+            initialSortColumn: this._sortBy,
+            initialSortDirection: this._sortDirection
+        });
+    }
+
+    @bind
+    rowClassName({index}: any) {
+        if (_.includes(this.props.selectedRows, index)) {
+            return styles.highlightedRow;
+        } else if (index < 0) {
+            return styles.headerRow;
+        } else {
+            return index % 2 === 0 ? styles.evenRow : styles.oddRow;
+        }
+    }
+
+    @bind
+    rowGetter({index}: any) {
+        return this._store.dataStore.sortedFilteredData[index];
+    }
+
+    @bind
+    getColumn(columnKey: string) {
+        return _.keyBy(this.props.columns, column => column.name)[columnKey];
+    }
+
+    @bind
+    @action
+    sort({sortBy}: any) {
+        this._store.defaultHeaderClick(this.getColumn(sortBy));
+        this._sortBy = sortBy;
+        this._sortDirection = this._store.dataStore.sortAscending ? 'asc' as 'asc' : 'desc' as 'desc';
+    }
+
+    @bind
+    @action
+    onFilterTextChange() {
+        let searchTimeout: number | null = null;
+        return (evt: any) => {
+            if (searchTimeout !== null) {
+                window.clearTimeout(searchTimeout);
+                searchTimeout = null;
+            }
+
+            const filterValue = evt.currentTarget.value;
+            searchTimeout = window.setTimeout(() => {
+                this._store.setFilterString(filterValue);
+            }, 400);
+        };
+    }
+
+    @bind
+    afterSelectingRows() {
+        if (_.isFunction(this.props.afterSelectingRows)) {
+            this.props.afterSelectingRows();
+        }
+    }
+
+    public render() {
+        return (
+            <div className={styles.studyViewTablesTable}>
+                <RVTable
+                    width={this.props.width!}
+                    height={this.props.height!}
+                    headerHeight={20}
+                    rowHeight={25}
+                    rowCount={this._store.dataStore.sortedFilteredData.length}
+                    rowGetter={this.rowGetter}
+                    className={styles.studyViewTablesBody}
+                    rowClassName={this.rowClassName}
+                    headerClassName={styles.headerColumn}
+                    sort={this.sort}
+                    sortDirection={RVSDTtoStrType[this._sortDirection]}
+                    sortBy={this._sortBy}
+                >
+
+                    {
+                        this.props.columns.map(column => {
+                            return <RVColumn label={column.name} dataKey={column.name} width={Number(column.width)}
+                                             cellRenderer={function (props: TableCellProps) {
+                                                 return column.render(props.rowData);
+                                             }.bind(this)}/>;
```

Comment:
Is there a specific reason here to use a `function` keyword and manually bind instead of using an arrow function like this:
`(props: TableCellProps) => column.render(props.rowData)`